ngsi_adapter/src/test/acceptance/features/component/send_data/send_data_api_resource.feature

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+Feature: Sending probe data
+  As a monitoring element (probe) user
+  I want to be able to transform monitoring data from probes to NGSI context attributes
+  so that I can constantly check the status and performance of the cloud infrastructure using NGSI Context Broker.
+
+
+  @happy_path
+  Scenario: Probe sends valid raw data using an existing and well-formed parser
+    Given the parser "qa_parser"
+    And   the monitored resource with id "qa:192.168.1.2" and type "host"
+    When  I send raw data according to the selected parser
+    Then  the response status code is "200"
+
+
+  Scenario: Probe sends valid raw data using a not existing parser
+    Given the parser "not_existing"
+    And   the monitored resource with id "qa:1234567890" and type "host"
+    When  I send raw data according to the selected parser
+    Then  the response status code is "404"
+
+
+  Scenario Outline: Probe sends valid raw data using an existing and well-formed parser, with valid probe ID values.
```

Comment:
**id** denoted the entity being monitored, *not* the probe. I guess here you mean "entity ID"